web-app/components/pages/login/login.jsx

```diff
@@ -82,6 +83,32 @@ export const Login = () => {
         },
         onSubmit: (values) => {
             console.log(JSON.stringify(values, null, 2));
+            axios({
+                method: 'post',
+                url: 'http://localhost:5000/api/authentication/login-attempt',
+                data: {
+                    email: values.email,
+                    password: values.password,
+                },
+            })
+                .then(function (response) {
+                    console.log(response.data);
+                    const token = response.data.token;
+                    if (token) {
+                        localStorage.setItem('token', token);
```

Comment:
think about a very unique name for the `local storage` key rather than `token` and please not to hard code it, store it somewhere in a utils file or constants file